Cannot find module 'ts-fsrs' when update to version 5

I got error when update  "ts-fsrs": "^5.0.0". It works for version 4.7.1
```
   return new TSError(diagnosticText, diagnosticCodes, diagnostics);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/fsrs.service.ts:1:124 - error TS2307: Cannot find module 'ts-fsrs' or its corresponding type declarations.

1 import { Card, createEmptyCard, FSRS, fsrs, generatorParameters, Grade, Rating, RecordLog, RecordLogItem, ReviewLog } from 'ts-fsrs';
```

[Reproduce repo](https://github.com/docaohuynhcse/fastify-fsrs)

--- a/jsr.json
+++ b/jsr.json
@@ -6,7 +6,7 @@
         "./constants": "./src/fsrs/constants.ts"
     },
     "name": "@fsrs/ts-fsrs",
-    "version": "5.0.0",
+    "version": "5.0.1",
     "license": "MIT",
     "publish": {
         "include": [

--- a/package.json
+++ b/package.json
@@ -1,7 +1,10 @@
 {
   "name": "ts-fsrs",
-  "version": "5.0.0",
+  "version": "5.0.1",
   "description": "ts-fsrs is a versatile package written in TypeScript that supports ES modules, CommonJS, and UMD. It implements the Free Spaced Repetition Scheduler (FSRS) algorithm, enabling developers to integrate FSRS into their flashcard applications to enhance the user learning experience.",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
   "exports": {
     ".": {
       "require": "./dist/index.cjs",

